### PR TITLE
Consistent worker iterations

### PR DESCRIPTION
**Pull request by Marcial1234**
### Description

Reduce length of web-workers event handler definition

### Motivation

Original author's motivation in https://github.com/mdn/dom-examples/pull/170:
```
I guess it's more readable.
```

### Related issues and pull requests

Original PR moving to iteration: https://github.com/mdn/dom-examples/pull/170
General PR to move all DOM Examples to this pattern: https://github.com/mdn/dom-examples/pull/284

### Extra Details

We can further reduce characters with `function ()` => `() =>` if desired.

### CODE REVIEW

**Comment by Marcial1234** [comment]
The previous Bot's suggestion seems to be unrelated/out-of-scope for this change.

File: files/en-us/web/api/messageevent/index.md [hunk 1]
```diff
@@ -69,15 +69,12 @@ myWorker.port.start();
...
-  myWorker.port.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
-
-second.onchange = () => {
-  myWorker.port.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
+[first, second].forEach((input) => {
+  input.onchange = function () {
```
Comment (bsmth) [inline files/en-us/web/api/messageevent/index.md:73]:
```suggestion
  input.onchange = () => {
```

File: files/en-us/web/api/sharedworker/index.md [hunk 2]
```diff
@@ -60,15 +60,12 @@ When the port is started, both scripts post messages to the worker and handle me
...
-  myWorker.port.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
-
-second.onchange = () => {
-  myWorker.port.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
+[first, second].forEach((input) => {
+  input.onchange = function () {
```
Comment (bsmth) [inline files/en-us/web/api/sharedworker/index.md:64]:
```suggestion
  input.onchange = () => {
```

File: files/en-us/web/api/sharedworker/sharedworker/index.md [hunk 3]
```diff
@@ -82,15 +82,12 @@ const myWorker = new SharedWorker("worker.js");
...
-  myWorker.port.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
-
-second.onchange = () => {
-  myWorker.port.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
+[first, second].forEach((input) => {
+  input.onchange = function () {
```
Comment (bsmth) [inline files/en-us/web/api/sharedworker/sharedworker/index.md:86]:
```suggestion
  input.onchange = () => {
```

File: files/en-us/web/api/web_workers_api/using_web_workers/index.md [hunk 4]
```diff
@@ -66,15 +66,12 @@ const myWorker = new Worker("worker.js");
...
-  myWorker.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
-
-second.onchange = () => {
-  myWorker.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
+[first, second].forEach((input) => {
+  input.onchange = function () {
```
Comment (bsmth) [inline files/en-us/web/api/web_workers_api/using_web_workers/index.md:70]:
```suggestion
  input.onchange = () => {
```

File: files/en-us/web/api/worker/postmessage/index.md [hunk 5]
```diff
@@ -48,15 +48,12 @@ The following code snippet shows the creation of a {{domxref("Worker")}} object
...
-  myWorker.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
-
-second.onchange = () => {
-  myWorker.postMessage([first.value, second.value]);
-  console.log("Message posted to worker");
-};
+[first, second].forEach((input) => {
+  input.onchange = function () {
```
Comment (bsmth) [inline files/en-us/web/api/worker/postmessage/index.md:52]:
```suggestion
  input.onchange = () => {
```

**Comment by bsmth** [comment]
Nice, thanks! Let's use the arrow functions, I think it was a leftover in the dom-examples repo, so we can be consistent here

**Review by bsmth — APPROVED** [review]
I think it's all looking good, thanks a lot!

**Pull request merged**